setup/kubernetes/k3s.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "k3s"
+description: Set up a Coder deployment with a Ubuntu + k3s
+---
+
+k3s is a lightweight Kubernetes distribution that works well for single-node or
```

Comment:
This is a great contribution, thank you!

I think we should indicate that k3s is not officially supported by Coder at this time, or make it part of our supported/tested configurations. What do you think?

Reply:
I am happy to add a note it not officially supported, but don't want to discourage bug reports in the Slack, so I might even link to it for "community support."

However, I personally think this is something we should consider supporting/testing depending on usage we see from the guide. I will also be sure to dogfood with it.

Reply:
I added a disclaimer✅